scripts/buildRpm.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
```

Comment:
@KjellKod I was initially retrieving the branch version using git. What happened was when you branched to a branch that was not a neat version #, the build would not work. I currently have it hardcoded down below to 4.1.4.